spec/system/managing_users_spec.rb

```diff
@@ -129,3 +133,7 @@
     end
   end
 end
+
+def secure_password
+  SecureRandom.random_bytes(256).chars.select { |b| b.ord > 32 && b.ord < 0x7f }.join[0, 128]
+end
```

Comment:
Was going to suggest to perhaps move this into a helper under `spec/support` - maybe if we have to repeat this once more though

Reply:
I'm not even sure if this is the best way to generate a password, but the problem I had was that if the random passwords are even slightly borderline, then given the number of test runs it'll fail an unacceptable number of times.